packages/manager/apps/container/src/payment-modal/PaymentModal.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useState } from 'react';
+import { Modal } from 'react-bootstrap';
+import { useTranslation } from 'react-i18next';
+import { useShell } from '@/context';
+import { fetchIcebergV6 } from '@ovh-ux/manager-core-api';
+import { useQuery } from '@tanstack/react-query';
+import { PAYMENT_ALERTS } from './constants';
+import './styles.scss';
+
+interface IPaymentMethod {
+  icon?: any;
+  label: string;
+  status: string;
+  default: boolean;
+  oneclick: boolean;
+  lastUpdate: string;
+  description: string;
+  integration: string;
+  paymentType: string;
+  creationDate: string;
+  paymentMeanId: number;
+  expirationDate?: string;
+  paymentSubType: string;
+  paymentMethodId: number;
+}
+
+const computeAlert = (paymentMethods: IPaymentMethod[]): string => {
+  const hasDefaultPaymentMethod: IPaymentMethod = paymentMethods?.find(currentPaymentMethod => currentPaymentMethod.default);
+  if (!hasDefaultPaymentMethod) {
+    return PAYMENT_ALERTS.NO_DEFAULT;
+  }
+  const currentCreditCard: IPaymentMethod = paymentMethods?.find(currentPaymentMethod => currentPaymentMethod.paymentType === 'CREDIT_CARD');
+  if (currentCreditCard) {
+    const creditCardExpirationDate: Date = new Date(currentCreditCard.expirationDate);
```

Comment:
```suggestion
    const creditCardExpirationDate = new Date(currentCreditCard.expirationDate);
```

No need to write type everytime